romana-install/romana-setup

```diff
@@ -49,14 +56,53 @@ if (( $# > 0 )); then
 				usage
 				exit 0
 				;;
-			-t|--target)
+			-n|--name)
+				case "$2" in
+					[a-zA-Z][a-zA-Z0-9]*)
+						stack_name="$2"
+						shift 2
+						;;
+					*)
+						echo "Invalid stack name: '$2' (must start with a letter, and contain only letters and digits)"
+						exit 1
+						;;
+				esac
+				;;
+			-p|--platform)
 				case "$2" in
-					aws|vagrant)
-						target="$2"
+					aws)
+						platform="aws"
+						required+=( "aws" )
+						shift 2
+						;;
+					vagrant)
+						platform="$2"
```

Comment:
For consistency's sake, could we do it the same was as with 'aws' and say: platform="vagrant" ?

Or would it be even easier to say (pseudo-code):

```
if $2 == "aws" or $2 == "vagrant":
    platform=$2
    required+= ( $2 )
else:
    unknown platform
```